temp/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.data
+*.exe
```

Comment:
Why do you add `temp` directory, @dirtysalt ? Could you revert this `temp` directory?

Reply:
Same issue, sorry.